clients/src/main/java/org/apache/kafka/common/record/DefaultRecordBatch.java

```diff
@@ -393,6 +395,24 @@ private static byte computeAttributes(CompressionType type, TimestampType timest
         return attributes;
     }
 
+    public static void writeEmptyHeader(ByteBuffer buffer,
```

Comment:
#3406 uses this in non-test code. We will merge that soon as well, so it seems like this way will cause less churn.